client/pom.xml

```diff
@@ -39,8 +39,11 @@
       </plugin>
 
       <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
+       
+		<groupId>org.jacoco</groupId>
+		<artifactId>jacoco-maven-plugin</artifactId>
+		<version>0.7.5.201505241946</version>
```

Comment:
Why have you added the version into the child poms, was there a specific reason or can it be put into the parent pom?